src/app/features/home/home.page.ts

```diff
@@ -197,9 +198,17 @@ export class HomePage {
 
   // eslint-disable-next-line class-methods-use-this
   async openCaptureClub() {
-    return Browser.open({
-      url: 'https://captureclub.cc/',
-      toolbarColor: '#564dfc',
-    });
+    return this.diaBackendAuthService.token$
+      .pipe(
+        first(),
+        switchMap(token => {
+          return Browser.open({
+            url: `https://captureclub.cc/?token=${token}`,
+            toolbarColor: '#564dfc',
+          });
+        }),
```

Comment:
```suggestion
        switchMap(token =>
          Browser.open({
            url: `https://captureclub.cc/?token=${token}`,
            toolbarColor: '#564dfc',
          })
        ),
```

Could use implicit return for single line arrow function.